install.sh

```
@@ -1533,7 +1663,8 @@ configure_web_port() {
                     print_message "✅ Port $WEB_PORT is available" "$GREEN"
                     break
                 else
-                    print_message "❌ Port $custom_port is also in use. Please try another port." "$RED"
+                    local process_info=$(get_port_process_info "$custom_port")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)